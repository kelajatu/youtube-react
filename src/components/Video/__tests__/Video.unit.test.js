import React from "react";
import { Video } from "../Video";
import { shallow } from "enzyme";

it("renders video component", () => {
  const wrapper = shallow(<Video id="HAuXJVI_bUs" />);
  expect(wrapper).toMatchSnapshot();
});

it("renders null if id invideo component no specified", () => {
  const wrapper = shallow(<Video />);
  expect(wrapper).toMatchSnapshot();
});
