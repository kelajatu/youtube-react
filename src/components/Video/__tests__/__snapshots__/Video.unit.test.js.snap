// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders null if id invideo component no specified 1`] = `""`;

exports[`renders video component 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/HAuXJVI_bUs?autoplay=0"
      title="video"
    />
  </div>
</div>
`;
